<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QMainWindow" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiAlign - CISA</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/technorati_48.png</normaloff>images/technorati_48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>671</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>inpute sequence ADN</string>
        </attribute>
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>661</width>
           <height>341</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Alignement</string>
        </attribute>
        <widget class="QTextEdit" name="textEdit_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>661</width>
           <height>341</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Graphe</string>
        </attribute>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>671</width>
           <height>341</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>671</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="suivant">
       <property name="text">
        <string>Suivant </string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/arrow_right_green_48.png</normaloff>images/arrow_right_green_48.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNeuveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrie"/>
    <addaction name="actionEnregistrie_sous"/>
    <addaction name="separator"/>
    <addaction name="actionImpemer"/>
    <addaction name="actionPrint_View"/>
    <addaction name="separator"/>
    <addaction name="actionQuiter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPast"/>
    <addaction name="actionDelet"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_all"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
   </widget>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAlignement">
    <property name="title">
     <string>Alignement tool</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="separator"/>
    <addaction name="actionGraph"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAffichage"/>
   <addaction name="menuAlignement"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNeuveau"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnregistrie"/>
   <addaction name="actionEnregistrie_sous"/>
   <addaction name="separator"/>
   <addaction name="actionImpemer"/>
   <addaction name="actionPrint_View"/>
  </widget>
  <widget class="QToolBar" name="toolBarEdit">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPast"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
  </widget>
  <widget class="QToolBar" name="toolBarAlign">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionGraph"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionNeuveau">
   <property name="icon">
    <iconset>
     <normaloff>images/new.png</normaloff>images/new.png</iconset>
   </property>
   <property name="text">
    <string>Neuveau</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset>
     <normaloff>images/ouvrir.png</normaloff>images/ouvrir.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionEnregistrie">
   <property name="icon">
    <iconset>
     <normaloff>images/disquette.png</normaloff>images/disquette.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrie</string>
   </property>
  </action>
  <action name="actionEnregistrie_sous">
   <property name="icon">
    <iconset>
     <normaloff>images/saveAs.PNG</normaloff>images/saveAs.PNG</iconset>
   </property>
   <property name="text">
    <string>Enregistrie sous</string>
   </property>
  </action>
  <action name="actionImpemer">
   <property name="icon">
    <iconset>
     <normaloff>images/print.png</normaloff>images/print.png</iconset>
   </property>
   <property name="text">
    <string>impemer</string>
   </property>
  </action>
  <action name="actionPrint_View">
   <property name="icon">
    <iconset>
     <normaloff>images/search.ico</normaloff>images/search.ico</iconset>
   </property>
   <property name="text">
    <string>Print View</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionQuiter">
   <property name="text">
    <string>Quiter</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>images/undo.png</normaloff>images/undo.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>images/redo.png</normaloff>images/redo.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>images/couper.svg.png</normaloff>images/couper.svg.png</iconset>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>images/copy.png</normaloff>images/copy.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
  </action>
  <action name="actionPast">
   <property name="icon">
    <iconset>
     <normaloff>images/coller.svg.png</normaloff>images/coller.svg.png</iconset>
   </property>
   <property name="text">
    <string>past</string>
   </property>
  </action>
  <action name="actionDelet">
   <property name="icon">
    <iconset>
     <normaloff>images/cross_48.png</normaloff>images/cross_48.png</iconset>
   </property>
   <property name="text">
    <string>delet</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>select all</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset>
     <normaloff>images/search1.png</normaloff>images/search1.png</iconset>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>images/wmp 11-05.png</normaloff>images/wmp 11-05.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionGraph">
   <property name="icon">
    <iconset>
     <normaloff>images/bold.png</normaloff>images/bold.png</iconset>
   </property>
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
